test/certs/generate.sh

```diff
@@ -3,32 +3,50 @@ set -e
 
 cd "$(realpath -- $(dirname -- "$0"))"
 
-ipki() (
-  # Check that `minica` is installed
-  command -v minica >/dev/null 2>&1 || {
-    echo >&2 "No 'minica' command available.";
-    echo >&2 "Check your GOPATH and run: 'go install github.com/jsha/minica@latest'.";
-    exit 1;
-  }
+# Check that `minica` is installed
+command -v minica >/dev/null 2>&1 || {
+  echo >&2 "No 'minica' command available.";
+  echo >&2 "Check your GOPATH and run: 'go install github.com/jsha/minica@latest'.";
+  exit 1;
+}
 
+ipki() (
   # Minica generates everything in-place, so we need to cd into the subdirectory.
   # This function executes in a subshell, so this cd does not affect the parent
   # script.
   mkdir ipki
   cd ipki
 
-  # Used by challtestsrv to negotiate DoH handshakes.
-  # TODO: Move this out of the ipki directory.
-  # This also creates the issuer key, so the loops below can run in the
-  # background without competing over who gets to create it.
+  # Create a generic cert which can be used by our test-only services (like
+  # mail-test-srv) that aren't sophisticated enough to present a different name.
+  # This first invocation also creates the issuer key, so the loops below can
+  # run in the background without racing to create it.
+  minica -domains localhost
```

Comment:
This one doesn't get backgrounded because it's implicit job is to create the minica.pem and minica.key files that the rest of the minica calls will reference.